README.md

```diff
@@ -98,6 +102,8 @@ ok 2 PhantomJS 1.9 - Exam Partition #3 - some other other test
 ok 3 PhantomJS 1.9 - Exam Partition #2 - some other test
 ```
 
+You can also combine the `parallel` option with the `partition` option to split tests, and then recombine partitions into parallel runs. This would, for example, allow you to run tests in multiple CI containers and have each CI container parallelize its list of tests.
+
```

Comment:
Can we add another code example here of how to use the partition option with parallel?